### PR TITLE
Remove server upgrade shutdown restart ops in ui

### PR DESCRIPTION
**Pull request by pravinrajr9**
Removal of OSC server shutdown, restart and upgrading operations from OSC Web UI

## Description
The OSC server shutdown, restart and upgrading features have been removed from OSC UI code

## How Has This Been Tested?
This has been tested manually 

## Screenshots (if appropriate):
image

image

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [Code style guidelines](https://github.com/opensecuritycontroller/community/blob/master/development/eclipse.md) of this project
- [ ] My unit test follow the [Unit test guidelines](https://github.com/opensecuritycontroller/community/blob/master/development/unit_test_guidelines.md)
- [ ] Unit test coverage percentage is maintained(increased/remains the same but not decreased)

### CODE REVIEW

**Comment by pravinrajr9** [comment]
@arvindn05 please review

**Comment by pravinrajr9** [comment]
@arvindn05 I made changes as you said.  Please review

File: osc-server/src/main/java/org/osc/core/broker/util/ServerUtil.java [hunk 1]
```diff
@@ -480,230 +472,6 @@ public void run() {
...
-            backupServerJar.delete();
-            successRename = serverJar.renameTo(backupServerJar);
-            if (!successRename) {
-                // File was not successfully renamed
-                throw new IOException("Fail to rename original server file.");
-            }
-
-            // invoke the upgrade script
-            Version currentVersion = VersionUtil.getVersion();
-            String script = tmpUpgrade + "/opt/vmidc/bin/scripts/upgrade.sh";
```
Comment (arvindn05) [inline osc-server/src/main/java/org/osc/core/broker/util/ServerUtil.java]: this script needs to be remove...not sure if its in 2 places.

Comment (pravinrajr9) [reply osc-server/src/main/java/org/osc/core/broker/util/ServerUtil.java]: yes @arvindn05, I have deleted it now, It is located in only one place. @arvindn05, @sudhirappaji  Please review

**Pull request merged**